pkg/apis/ibmcloudprovider/v1beta1/ibmcloudproviderconfig_types.go

```diff
@@ -48,6 +48,9 @@ type IBMCloudMachineProviderSpec struct {
 	// Example: bx2-8x32 (8 vCPUs, 32 GB RAM)
 	Profile string `json:"profile"`
 
+	// Dedicated host
```

Comment:
If we could expand on this comment before merging to give a bit more detail about the feature that would be useful, these descriptions end up in the docs